modules/KoreBuild.Tasks/module.targets

```diff
@@ -9,6 +8,18 @@
     <DependencyVersionsFile Condition="'$(DependencyVersionsFile)' == ''">$(RepositoryRoot)build\dependencies.props</DependencyVersionsFile>
   </PropertyGroup>
 
+  <Target Name="EnableSkipStrongNames" Inputs="$(SkipStrongNamesXml)">
+    <Error Text="The parameter 'SkipStrongNamesXml' was not set." Condition="'$(SkipStrongNamesXml)' == ''" />
```

Comment:
We'd want to include a file by default (See https://github.com/aspnet/AspNetWebStack/blob/master/tools/SkipStrongNames.xml). You could leave it empty now but have the right structure. The ideal workflow would be for us to do a regular build and this target kicks in without any additional configuration.